daemons/pacemakerd/pacemakerd.c

```diff
@@ -330,7 +330,10 @@ main(int argc, char **argv)
     }
 
     pcmk_register_ipc_callback(old_instance, pacemakerd_event_cb, NULL);
-    rc = pcmk_connect_ipc(old_instance, pcmk_ipc_dispatch_sync);
+    rc = pcmk__connect_ipc(old_instance, pcmk_ipc_dispatch_sync, 2);
+    if (rc != pcmk_rc_ok) {
+        crm_debug("No existing launcher instance found: %s", pcmk_rc_str(rc));
```

Comment:
optionally, could use `pcmk_ipc_name()` for these messages (here and below in this commit), to avoid hard-coding strings. Unlikely that we'll change them again but who knows

Reply:
good point